### PR TITLE
Update jquery-countdown autoupdate config with v2.1.0

### PR DESCRIPTION
**Pull request by sashberd**
Using web-minify-helper tool for minifying
v2.1.0 was added manually from git repo

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sashberd** [comment]
@PeterDaveHello PR was updated. FYR

File: ajax/libs/jquery-countdown/package.json [hunk 1]
```diff
@@ -30,6 +30,12 @@
         "files": [
           "jquery.+(plugin|countdown)*.+(js|css)"
         ]
+      },
+      {
+        "basePath": "dist",
+        "files": [
+          "**/(css|js)/jquery.+(plugin|countdown)*.+(js|css)"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-countdown/package.json]: `(css|js)` -> `+(css|js)`

**Comment by sashberd** [comment]
@PeterDaveHello Updated

**Comment by sashberd** [comment]
@PeterDaveHello I have no idea how this happens, my apologise, the PR was updated. FYR

**Comment by PeterDaveHello** [comment]
@sashberd please use request review next time ;)

**Comment by sashberd** [comment]
@PeterDaveHello The autoupdate config was updated, now not fetched files are fine. The commit message was also updated.  FYR

**Comment by sashberd** [comment]
@PeterDaveHello Done

**Comment by PeterDaveHello** [comment]
Not sure the purpose of the png & gif images under `img/`, looks not referenced in css/js file, I'll merge this one now, and it'll be even better to figure it out :+1:

**Pull request merged**